### ISSUE TITLE
different loss when running get_preds

### ISSUE THREAD

**Issue by radekosmulski** [fastai/fastai2#38]
Thank you for fixing the issue with `get_preds` and `language_model_learner`! 😊

I am now seeing this problem where the loss is different when running `get_preds` vs when I run the calculation manually. Everything is fine during training.

![image](https://user-images.githubusercontent.com/2444926/71810555-f13e0500-3072-11ea-8466-06e123888027.png)

**Comment by sgugger** [comment]
Don't forget that `Learner.get_preds` applies the last activation (here softmax) so you are not comparing apple to apple (which is why the accuracy is more or less the same since softmax doesn't change the order).
